src/Compilers/CSharp/Portable/Symbols/Source/ParameterHelpers.cs

```diff
@@ -409,8 +409,11 @@ private static void EnsureNullableAttributeExists(CSharpCompilation compilation,
             BaseParameterSyntax parameter,
             BindingDiagnosticBag diagnostics,
             bool parsingFunctionPointerParams,
-            bool parsingLambdaParams)
+            bool parsingLambdaParams,
+            bool parsingAnonymousMethod)
```

Comment:
If we're splitting these up, I wouldn't expect callers to pass true for both `parsingLambdaParams` and `parsingAnonymousMethod`. I'd expect it would be one or the other.

Reply:
That seems better, thanks. Changed: a6d8976aeef845a20df6b76790e0485a182ac055